src/find_datalad_repos/__main__.py

```diff
@@ -27,30 +28,42 @@ def set_mode(
     default=logging.INFO,
     help="Set logging level  [default: INFO]",
 )
+@click.option(
+    "--atris",
+    flag_value=RepoHost.ATRIS,
```

Comment:
since RepoHost is now introduced, could there just be a single `--repo-hosts` or `--hosts` which would take multiple values of RepoHost values?  That would avoid necessity for CLI modification for every host addition?

Also shortcut `*` could be introduced  to signal "all known" and thus avoid necessity to modify CI job if we keep indexing all "registered"?